exercises/complex-numbers/canonical-data.json

```diff
@@ -242,8 +244,8 @@
       ],
       "description": "Euler's identity/formula",
       "property": "exp",
-      "input": "pi*i",
-      "expected": "-1"
+      "input": [0, "pi"],
```

Comment:
I'm not happy with this but I can't think of a better way without using an approximation of π.